helm/defectdojo/values.yaml

```diff
@@ -109,6 +109,39 @@ celery:
       # concurrency: 8
       # prefetch_multiplier: 128
 
+  # A list of extra volumes to mount. This
```

Comment:
Would it make sense to copy this whole help into a separate file and reference here, especially since it is put in twice?

Reply:
It is not the same, because for celery there is single container, I wanted to follow practice to have explanation together with values.